src/base-apis.js

```diff
@@ -1889,6 +1889,44 @@ MatrixBaseApis.prototype.lookupThreePid = async function(
     }
 };
 
+/**
+ * Looks up the public Matrix ID mappings for multiple 3PIDs.
+ *
+ * @param {array} query Array of arrays containing [medium, address]
```

Comment:
the type could perhaps be more explicit like: `Array.<Array.<{string, string}>>`

Reply:
Thanks, updated type.